pubsub/jetstream/jetstream.go

```diff
@@ -24,6 +24,11 @@ import (
 	"github.com/dapr/kit/retry"
 )
 
+const (
+	// MessageIDMetadataKey defines the metadata key for the message id.
```

Comment:
@NickLarsenNZ isn't this unused?

Reply:
Good spotting. I'll remove it, and rebase too.